docs/prerequisites.md

```diff
@@ -27,43 +38,100 @@ Additional AWS credentials are needed for the operator to be successfully instal
 
 ### For STS clusters
 
-1. [Extract and prepare the `ccoctl` binary](https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts)
+#### Option 1. Using `ccoctl`
 
-2. Create AWS Load Balancer Operator's namespace:
-    ```bash
-    oc create namespace aws-load-balancer-operator
+1. [Extract and prepare the `ccoctl`
+   binary](https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts)
+
+2. Create AWS Load Balancer Operator's namespace: ```bash oc create namespace
+   aws-load-balancer-operator ```
+
+3. Use the `ccoctl` tool to process the operator's `CredentialsRequest` objects
+   needed to bootstrap the operator:
+
+    ```bash curl --create-dirs -o <path-to-credrequests-dir>/cr.yaml
+    https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
+    ccoctl aws create-iam-roles \ --name <name> --region=<aws_region> \
+    --credentials-requests-dir=<path-to-credrequests-dir> \
+    --identity-provider-arn <oidc-arn> ```
+
+    For each `CredentialsRequest` object, `ccoctl` creates an IAM role with a
+    trust policy that is tied to the specified OIDC identity provider, and
+    permissions policy as defined in each `CredentialsRequest` object. This
+    also generates a set of secrets in a **manifests** directory that is
+    required by the **aws-load-balancer-operator**.
+
+4. Apply the secrets to your cluster:
+
+    ```bash ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {} ```
+
+5. Verify that the operator's credentials secret is created:
+
+    ```bash oc -n aws-load-balancer-operator get secret
+    aws-load-balancer-operator -o json | jq -r '.data.credentials' | base64 -d
+    [default] sts_regional_endpoints = regional role_arn =
+    arn:aws:iam::999999999999:role/aws-load-balancer-operator-aws-load-balancer-operator
+    web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
     ```
 
-3. Use the `ccoctl` tool to process the operator's `CredentialsRequest` objects needed to bootstrap the operator:
+#### Option 2. Using AWS CLI
```

Comment:
Any comments I made in install.md - same for this file, where applicable.

Reply:
Done.